lib/registration.pm

```diff
@@ -185,7 +185,18 @@ sub fill_in_registration_data {
                     };
                     next;
                 }
-                elsif (match_has_tag('inst-addon') || match_has_tag('yast_scc-pkgtoinstall')) {
+                elsif (match_has_tag('yast_scc-pkgtoinstall')) {
+                    send_key 'alt-a';
+                    assert_screen 'automatic-changes';
+                    send_key 'alt-o';
+                    if (check_screen 'unsupported-packages', 5) {
+                        send_key 'alt-o';
+                    }
```

Comment:
you know I don't like timeouts on check_screen which are not 0, so what about

```perl
wait_screen_change { send_key 'alt-o' };
send_key 'alt-o' if check_screen('unsupported-packages', 0);
```
?